### ISSUE TITLE
Support for Powerups

### ISSUE THREAD

**Issue by shuffgy** [IS2511/ChatIS#26]
Twitch now has "Powerups". These are special messages you can use with bits. There are new events for it also.

[Link to blogpost](https://blog.twitch.tv/en/2024/06/12/introducing-power-ups-unleash-special-effects-with-bits/)

Would be awesome to somehow support these (bigger emote, blinky message) or at least show the messages.

**Comment by IS2511** [comment]
The power-up "messages" are actually [EventSub API](https://dev.twitch.tv/docs/eventsub/) events, which ChatIS doesn't support at all because it's a totally different system from the Twitch chat itself. I'm not even sure it's possible to publicly subscribe to the needed events (like [`channel.chat.message`](https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types/#channelchatmessage)), most require perms that are only available to authenticated apps and users. Requiring streamers to log-in in their OBS is a bad idea. Offloading the auth to my server and proxying the events is an even worse idea. The optimal approach would probably be to have a Twitch login on my website and give out chat URLs with a personal API token. Kinda unsafe, but probably the best UX I can offer, similar to what existing overlay platforms are doing.

TLDR: A pain in the ass to implement, huge UX changes required, probably a v3 endeavor.
